src/services/MapFeedbackService.js

```diff
@@ -3,39 +3,64 @@
  */
 /**
  * @typedef {Object} MapFeedback
- * @property {string} state the encoded state of the map
- * @property {string} category the category of this feedback message
- * @property {string} description the category of this feedback message
- * @property {string} [email] the email address of the editor of this feedback message
- * @property {string} [fileId] the id of the referenced geometry file of this feedback message
+ * @property {string} state The encoded state of the map
+ * @property {string} category The category of this feedback message
+ * @property {string} description The actual message of this feedback message
+ * @property {string} fileId The id of the referenced geometry file of this feedback message
+ * @property {string} [email] The email address of the editor of this feedback message
+ */
+
+import { bvvMapFeedbackCategoriesProvider, bvvMapFeedbackStorageProvider } from './provider/mapFeedbackStorage.provider';
+
+/**
+ * A function that stores a MapFeedback
+ * @async
+ * @param {module:services/MapFeedbackService~MapFeedback} mapFeedback
+ * @typedef {Function} mapFeedbackStorageProvider
+ * @throws {@link Error} when storing was not succesfull
```

Comment:
Was this intentionally not changed?
like the other changes to `...@throws ``Error``...`